pkg/driver/clients/grpc_client_test.go

```diff
@@ -98,8 +123,25 @@ func TestGRPCClientWithServerReachable(t *testing.T) {
 		record.ActualDuration == 0 ||
 		record.ActualMemoryUsage == 0 {
 
-		t.Error("Failed gRPC invocations.")
+		t.Error("Failed gRPC invocations for non vSwarm.")
 	}
+	newinvoker := CreateInvoker(cfgSwarm, nil, nil)
+
+	start = time.Now()
+	success, record = newinvoker.Invoke(&testFunction, &testRuntimeSpecs)
```

Comment:
This should fail. We deploy the same gRPC server as runs in the invitro images. It should fail to correctly respond to vSwarm invoker.